packages/components/field/src/js/field.controller.js

```diff
@@ -148,7 +148,8 @@ export default class FieldController {
     }
 
     this.checkAllErrors();
-    return this.invalid && (this.forceErrorDisplay || this.blurred || this.form.$submitted);
+    return this.invalid
+      && (this.form[this.currentErrorField].$modelValue || this.blurred || this.form.$submitted);
```

Comment:
**nitpick:** Condition should be updated to check if there is any value on the `input` element associated with this particular `oui-field`